### PR TITLE
Questions page and quick tips panel

### LINKED ISSUE #389: Questions page to various help links
As a user, when I click `Questions?` button in the transcription, I am taken to a page where I am given choices based on what I need help with. I can see three boxes that link out to: 
- Discussion forum: I want to talk about what I am finding 
- Help Center: I want to know how to do something
- Contact Us: I have a specific question to a Community manager 

This is a self guided routing page for users to determine how they would like to get help. 
Acceptance criteria: 
- create a new static page title `questions.md`
- take the styling of the guide cards in the Help center and link out to: History Hub, Help Center,  contact us page 

mymcards

### PR DESCRIPTION
**Pull request by elainekamlley**
Resolves #389 and #371 

Changes:
- Create questions page for users to route them to the correct space to answer their question
- Changed the link to question button to go to question page
- Added content to instructions panel

### CODE REVIEW

File: static-pages/questions.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <div class="card-body pxy-default">
+        <h3 class="card-title"><a href="/help-center/">Help Center &raquo;</a></h3>
+        <h5 class="card-text">Find guides to help you tag, transcribe, and review.</h5>
+      </div>
+    </div>
+  </div>
+  <div class="col-sm-4">
+    <div class="help-center-card card text center mx-default">
+      <div class="card-body pxy-default">
+        <h4 class="card-title"><a href="https://historyhub.history.gov/community/make-your-mark/">Discuss &raquo;</a></h4>
```
Comment (rstorey) [inline static-pages/questions.md]: need to add target="_blank" to the anchor tag for history hub

File: concordia/templates/transcriptions/asset_detail.html [hunk 2]
```diff
@@ -142,7 +142,7 @@ <h2>Tags</h2>
                     {% else %}
                         <p class="help-text primary-text">
                             Only registered users can tag.
-                            <a class="primary-text" href="/account/register/">Register</a> or <a class="primary-text" href="/account/login">Login</a>
+                            <a class="primary-text" href="/account/register/">Register</a> or <a class="primary-text" href="/accounts/login">Login</a>
```
Comment (acdha) [inline concordia/templates/transcriptions/asset_detail.html]: We should use URL reversing to make sure these paths are always correct. The login link also will now return them to this page:

```suggestion
                            <a class="primary-text" href="{% url 'registration_register' %}">Register</a> or <a class="primary-text" href="{% url 'login' %}?next={{ request.path|urlencode }}">Login</a>
```

File: static-pages/help-center.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ title: Help Center
         <img class="help-center-overlay" src="/static/img/help-center.jpg" alt="Placeholder Help Image">
           <div class="card-body pxy-default card-img-overlay">
             <h2 class="card-title text-white">Welcome to Make Your Mark!</h2>
-            <h5 class="card-text"> Here you will find a guide to help you get started. </h5>
+            <h5 class="card-text"> Here you will find a guide to help you get started.</h5>
```
Comment (acdha) [inline static-pages/help-center.md]:
```suggestion
            <h5 class="card-text">Here you will find a guide to help you get started.</h5>
```

File: static-pages/questions.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      <div class="card-body pxy-default">
+        <h3 class="card-title"><a href="/help-center/">Help Center &raquo;</a></h3>
+        <h5 class="card-text">Find guides to help you tag, transcribe, and review.</h5>
+      </div>
+    </div>
+  </div>
+  <div class="col-sm-4">
+    <div class="help-center-card card text center mx-default">
+      <div class="card-body pxy-default">
+        <h4 class="card-title"><a target="_blank" href="https://historyhub.history.gov/community/make-your-mark/">Discuss &raquo;</a></h4>
```
Comment (rstorey) [inline static-pages/questions.md]: These = signs look really weird, are they correct?

Comment (acdha) [reply static-pages/questions.md]: I think that's GitHub's parser choking on HTML-in-Markdown

Comment (acdha) [reply static-pages/questions.md]: The generated markup is valid and the resulting page looks correct, too.

**Pull request merged**